### ISSUE TITLE
HiPlot panel does not show anything

### ISSUE THREAD

**Issue by lemairecarl** [comet-ml/issue-tracking#448]
## Describe the Bug
The HiPlot panel does not show any experiment. I'm also wondering why it says there are 169 filters, but none are in the list (see screenshot below)

## Expected behavior
A parallel coordinates plot should appear.
Also, there should be some documentation about how to specify the target metric and the other metrics to display. The panel's code does not make it clear.

## Where is the issue?
- [ ] Comet Python SDK
- [x] Comet UI
- [ ] Third Party Integrations (Huggingface, TensorboardX, Pytorch Lighting etc)

## To Reproduce
1. Create a HiPlot panel

## Screenshots or GIFs

![Screenshot from 2021-11-24 15-04-22](https://user-images.githubusercontent.com/13444373/143306186-804afe1a-7b6e-4afc-8393-68db09e63436.png)

![Screenshot from 2021-11-24 14-59-13](https://user-images.githubusercontent.com/13444373/143305606-80b8a4c2-e130-46ef-9ebe-73458cd560cd.png)

## Link to Comet Project/Experiment
https://www.comet.ml/lemairecarl/microscopie/view/lzy4th53wbc8iEZCjpV3eh2kj/panels

**Comment by lemairecarl** [comment]
I just noticed that the number on the Filters tab is the number of experiments in the result of the filtering.
In the image above there is a filter applied, but there are still many experiments. Removing the filter does not help.

**Comment by DN6** [comment]
Hello @lemairecarl. I will take a look into this. Would it be possible to make your project public so that I can reproduce the issue?

**Comment by DN6** [comment]
Just following up here, would the built in Parallel Coordinates Chart work for your use case?

**Comment by lemairecarl** [comment]
The project is now public, sorry about that.

The built in Parallel Coordinates Chart has multiple problems. Two of them come to mind: (1) It's not possible to get a sense of how many lines go from a specific point to another specific point. (2) There is no interactivity.
Hiplot seems to be better on these points. Interactivity seems to make it more useful.

**Comment by lemairecarl** [comment]
Ah! Thanks for the info. It makes sense that the panel would work with Comet Optimizer. However it would have been nice to document somewhere that it works exclusively with it.

It's very nice of you to offer to work on supporting Optuna! I would indeed like to try HiPlot with my Optuna experiments.
Thanks!

**Comment by dsblank** [comment]
FYI, there are also some notes here on running a 3rd-party optimizer with Comet tools: https://www.comet.ml/docs/python-sdk/other-optimizers/

**Comment by lemairecarl** [comment]
Thanks @dsblank , that's helpful! By looking at that page, and the code of the HiPlot panel, I discovered that I needed to set the following 3 keys in the "Other" section (`experiment.log_other`, hyperparameters don't work): `optimizer_metric`, `optimizer_metric_value`, `optimizer_parameters`.

Now the HiPlot panel works! I'll work with that for now.

**Comment by lemairecarl** [comment]
I closed the issue since the main problem is solved (now I can use HiPlot), however it would still be appreciated if the panel could be improved to help working with other optimizers. A simple thing would be to add some documentation on which keys need to be set in the experiment for the panel to work. A more complicated thing would be to add some configuration to enable using experiments that did not log the 3 specific keys mentioned in my last comment.
